configure.ac

```diff
@@ -280,13 +280,13 @@ AS_CASE([$ENABLED_FIPS],
     [ready|v5-ready],[
         FIPS_VERSION="v5-ready"
         HAVE_FIPS_VERSION=5
-        HAVE_FIPS_VERSION_MINOR=2
+        HAVE_FIPS_VERSION_MINOR=3
```

Comment:
Yes, indeed the proximal motivation was to allow `--enable-fips=v5` and `--enable-fips=ready` to both build after the API change.

This is actually fixing an oversight associated with the FIPS 140-3 push.  Until now, we've always had a separate version for `-ready`, e.g. `v2` (cert3389) vs `v3` (`-ready`).  Wtih this adjustment, we've gotten back to that.

Note that we now have a handy scheme in place so that whenever a new FIPS codesnap is tagged (for an OE or for a whole new cert), everything is guaranteed to be in place to gate for it using the previous `-ready`/`-dev` version, which becomes the new tagged version.  We increment the minor digit to create the new `-ready`/`-dev` version going forward.

@ejohnstown does have a general rule in place that the major version of libwolfssl consistently matches the major version of FIPS.  They only differ in the minor digit(s).  This seems like the right idea to keep the confusion to a minimum.